scripts/makeupdates

```diff
@@ -223,8 +227,11 @@ def copy_updated_files(tag, updates, cwd, builddir):
             uidir = os.path.sep.join(dir_parts[g_idx+1:])
             install_to_dir(gitfile, os.path.join("usr/share/anaconda/ui/", uidir))
         elif gitfile.startswith('pyanaconda/'):
-            dirname = os.path.join(SITE_PACKAGES_PATH, os.path.dirname(gitfile))
-            install_to_dir(gitfile, dirname)
+            if "anaconda-webui" not in gitfile:
+                dirname = os.path.join(SITE_PACKAGES_PATH, os.path.dirname(gitfile))
+                install_to_dir(gitfile, dirname)
+            else:
+                install_to_dir("pyanaconda/ui/cockpit/anaconda-webui/dist", "usr/share/cockpit/anaconda-webui")
```

Comment:
You should be able to simplify this with the new location.

Reply:
Done.